zuul-core/src/main/java/com/netflix/zuul/Filter.java

```diff
@@ -46,6 +46,11 @@
      */
     FilterType type() default FilterType.INBOUND;
 
+    /**
+     * Category of the filter.
+     */
+    String category() default "zuul";
```

Comment:
This feels too open-ended as a string representation. 
At the very least, it should have a pre-defined set of categories in an enum etc.